pkg/kubelet/kubelet.go

```diff
@@ -1188,7 +1192,13 @@ func (kl *Kubelet) initializeModules() error {
 		return fmt.Errorf("Failed to start OOM watcher %v", err)
 	}
 
-	// Step 7: Start resource analyzer
+	// Step 7: Init Nvidia Manager. Do not need to return err until we use NVML instead.
+	// Only works when user give true to EnableNvidiaGPU
+	if kl.enableNvidiaGPU {
+		kl.nvidiaGPUManager.Init(kl.dockerClient)
```

Comment:
@vishh Could you help me implement this Stub? Thanks in advance. I just changed the "EnableNvidiaGPU" to "EnableExperimentalNvidiaGPU".